### ISSUE TITLE
Just an idea: customizable phrases

### ISSUE THREAD

**Issue by rojoweko** [quaap/PhoneFoneFun#1]
Maybe you like the idea of making the phrases the phone utters editable, so people can customize and localize the app for their toddlers themselves (furthermore the default phrases sound weird with a non-English TTS system ;-) ).

**Comment by quaap** [comment]
Good idea!  I had considered adding in some sort of editor or allowing text-file loading, but didn't have time to work it out.  I have a lot of family stuff happening now (moving, job changes, etc), but after that I might change things up a bit.

**Comment by treakiandroid** [comment]
Just read some (if existing) /sdcard/Android/com.quaap.phonefonefun/config/costsentence.ini that lookes like

[PhoneFoneFunCostTtsSentence]
Version=1
TtsLanguage=german
[Hash]
Pitchvalue=0.3
0=Hallo
1=schön das du anrufst
2=und wie gehts denn so
3=warst du heute schon draußen
4=Hänsel und Gretel verliefen sich im Wald
5=Es war so finster und auch so bitterkalt
6=sie kamen an ein Häuschen von Pfefferkuchen fein
7=Wer mag der Herr wohl von diesem Häuschen sein
8=Ja und wie geht es weiter noch mal gleich
9=Ja genau
10=Die beiden hatten Hunger und knuspert im Häuschen
11=Und auf einmal hörten sie eine Stimme einer alten Frau
12=Knusper knusper knäuschen wer knuspert an meinem Häuschen
13=Und die beiden antworteten
14=Der Wind der Wind das himmlische Kind
15=Oder so ähnlich
16=naja aber in der Kamera die Hexe und sperrte sie in den Stall
17=Und gaben ihnen essen
18=Und die Hexe war ziemlich kurzsichtig
19=Und weil die Hexe so schlecht sehen konnte wollte sie immer dass sie einen Finger durch die Tür stecken
20=damit konnte die Hexe dann gucken ob sie schon genug gemästet worden sind
21=Aber Hänsel und Gretel durchschaut dies
22=Und streckte nehmen einen alten Knochen hinaus
23=Aber aufgeschoben ist nicht aufgehoben wie du ja weißt
24=Irgendwann kam Hexe dann doch auf den Trichter dass sie mal genug gefressen hatten und Darm dann das eine Kind welche Rasse nicht mehr raus um es in den Ofen zu stecken
25=Weil die Hexe ja gerne leckere fette kleine Kinder ist
26=Oder frisst na ja egal
27=auf jeden Fall sollten dann die Kinder in den Ofen sollten, aber die Hexe wurde dann ausgetrickst und selber in den Ofen verfrachtet und dann war das Rohr zu und Hänsel und oder Gretel legte den Riegel vor und die Hexe starb.
28=und die Moral von der Geschicht:   wer anderen eine Grube gräbt fällt selbst hinein oder so ähnlich...
29=Ah ja und Hexen sind böse mittelalterliches Märchen Emanzipation war damals noch nicht so viel
30=Tschüschi, und ruf mich mal wieder an
[Star]
Pitchvalue=0.7
0=Moin
1=ganz gut, aber meine Mutter ist noch nicht zuhause...
2=aber ja doch, komme gerade mit dem rad von der schule
3=erzählt mir eine Geschichte
4=und weiter
5=ok
6=ach wirklich?
7=oh
8=Danke machs gut!
